packages/react-tooltip/src/components/Tooltip/useTooltip.tsx

```diff
@@ -175,7 +184,14 @@ export const useTooltip = (
     onBlur: useMergedCallbacks(childProps?.onBlur, onLeaveTrigger),
   };
 
-  const isServerSideRender = useIsSSR();
+  // If the target prop is not provided, attach targetRef to the trigger element's ref prop.
+  // Otherwise, use the target prop directly.
+  const childTargetRef = useMergedRefs(childProps?.ref, targetRef);
+  if (state.target === undefined) {
+    triggerProps.ref = childTargetRef;
+  } else {
+    targetRef.current = state.target;
```

Comment:
You can do this, but `usePopper` also accepts a `target` option that will take priority over the `targetRef` value

https://github.com/microsoft/fluentui/blob/8ebf2d422547bda60eb22eae2b12c3bd1339ec06/packages/react-positioning/src/types.ts#L93

Reply:
The way that usePopper resolves this is slightly different in the case that `target={null}` (which will be the case on the initial render for a ref that isn't yet resolved). Here's what usePopper does:
`const target = options.target || targetRef.current;`

So if `target={null}`, usePopper would fall back to `targetRef`, but the tooltip will still use `null`. I don't really know what's the best option; but I figured it would be better to consistently use the `target` prop if it is set, rather than ignoring it and falling back to the child ref.

What do you think about updating `usePopper` to this instead, so it only falls back to targetRef if target is undefined (not null)?
`const { target = targetRef.current } = options;`

Reply:
I guess this would occur if both `targetRef` and `target` contain valid elements ? it could trigger some interesting edge cases.

I like the idea of updating `usePopper`, the idea behind the `target` prop there was to avoid any weird ref assignments in components.